### PR TITLE
`azurerm_automation_software_update_configuration` - fix windows update classification use a comma separate string

### PR DESCRIPTION
**Pull request by wuxu92**
swagger definition as a comma-separated string

```
"includedUpdateClassifications": {
  "description": "Update classification included in the software update configuration. A comma separated string with required values",
  "type": "string",
  "enum": [
  "Unclassified",
```

https://github.com/Azure/azure-rest-api-specs/blob/99977890f7a2e8cf9b4e229f452c1af7b30184e4/specification/automation/resource-manager/Microsoft.Automation/stable/2019-06-01/softwareUpdateConfiguration.json#L400

```
--- PASS: TestAccSoftwareUpdateConfiguration_basic (166.01s)
--- PASS: TestAccSoftwareUpdateConfiguration_update (125.49s)
--- PASS: TestAccSoftwareUpdateConfiguration_windows (160.53s)
PASS

```

Fixes: https://github.com/hashicorp/terraform-provider-azurerm/issues/18537

### CODE REVIEW

File: internal/services/automation/automation_software_update_configuration.go [hunk 1]
```diff
@@ -512,12 +513,20 @@ func (m SoftwareUpdateConfigurationResource) Arguments() map[string]*pluginsdk.S
 					"classification_included": {
```
Comment (katbyte) [inline internal/services/automation/automation_software_update_configuration.go:513]: As this is supposed to be a list, and this property is misnamed can we deprecate this and create a new property `classifications_included` that takes a list and translates it to a CSV list for the api?

**Review by katbyte — CHANGES_REQUESTED** [review]

File: internal/services/automation/automation_software_update_configuration.go (hunk 1 above)
Comment (wuxu92) [reply internal/services/automation/automation_software_update_configuration.go:513]: I have tried with a new property but we have to make it compatible with the deprecated one or there will be diff in the `terraform plan`. So I have to set both properties as `Computed: true` to ignore the diff. I don't think it's a so good solution. Is there better practice for this scenario?

Comment (katbyte) [reply internal/services/automation/automation_software_update_configuration.go:513]: yes you will need to set both to computed, always set both, make them conflict, and the use the one that is set. that is the right way to deprecate.

**Review by katbyte — APPROVED** [review]
LGTM 🧄

**Pull request merged**